### PR TITLE
fix(app): Fix Error Recovery design inconsistencies

### PR DESCRIPTION
**Pull request by mjhuff**
Closes DQA tickets: [RQA-2983](https://opentrons.atlassian.net/browse/RQA-2983), [RQA-2988](https://opentrons.atlassian.net/browse/RQA-2988), [RQA-2984](https://opentrons.atlassian.net/browse/RQA-2984), [RQA-2991](https://opentrons.atlassian.net/browse/RQA-2991), [RQA-2996](https://opentrons.atlassian.net/browse/RQA-2996), [RQA-3041](https://opentrons.atlassian.net/browse/RQA-3041), [RQA-3043](https://opentrons.atlassian.net/browse/RQA-3043), [RQA-3039](https://opentrons.atlassian.net/browse/RQA-3039), and [RQA-3045](https://opentrons.atlassian.net/browse/RQA-3045)

# Overview

These are all DQA fixes for Error Recovery. The problems and proofs of fix are attached to each ticket. Commits correspond to the tickets. There are only a few commits worth discussion:

d2e085100cb2c4af7ac6c5da94225f6d4a4f41e8 - Authorship folks, @koji @jerader @ncdiehl11 , I noticed that the icon was never separated from the header text by the proper amount. Diffing the file, I'm confused if this has always been the case? It seems like something that would have been fixed in practice a long time ago, and I want to make sure that I'm not breaking all of our modals with a `header` here. Did we just refactor something recently and forget about this?

[af0f8e3](https://github.com/Opentrons/opentrons/pull/16030/commits/af0f8e32d4b7b3bba453c8b53b302c9dec064a47)  and [6d97506](https://github.com/Opentrons/opentrons/pull/16030/commits/6d97506fbbf8d2e1618d6e65569e9ffee1da32d8)- These are from figma discussions with design.

## Test Plan and Hands on Testing
- Every ticket contains proof of fix.

## Risk assessment
- low, outside of the modal fix above, d2e085100cb2c4af7ac6c5da94225f6d4a4f41e8. See description. 

[RQA-2983]: https://opentrons.atlassian.net/browse/RQA-2983?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-2988]: https://opentrons.atlassian.net/browse/RQA-2988?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-2984]: https://opentrons.atlassian.net/browse/RQA-2984?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-2991]: https://opentrons.atlassian.net/browse/RQA-2991?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-2996]: https://opentrons.atlassian.net/browse/RQA-2996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-3041]: https://opentrons.atlassian.net/browse/RQA-3041?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-3043]: https://opentrons.atlassian.net/browse/RQA-3043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-3045]: https://opentrons.atlassian.net/browse/RQA-3045?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[RQA-3039]: https://opentrons.atlassian.net/browse/RQA-3039?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by koji** [comment]
@mjhuff I think Seth pointed out that margin in Tamar's PR. I planned to do double-check but I was busy for PD stuff...
thank you for fixing that.

File: app/src/pages/RunSummary/index.tsx [hunk 1]
```diff
@@ -166,16 +167,20 @@ export function RunSummary(): JSX.Element {
...
     !cancelledWithoutRecovery &&
     ((runRecord?.data.errors != null && runRecord?.data.errors.length > 0) ||
       (commandErrorList != null && commandErrorList?.data.length > 0))
 
-  let headerText =
-    commandErrorList != null && commandErrorList.data.length > 0
-      ? t('run_completed_with_warnings_splash')
-      : t('run_completed_splash')
-  if (runStatus === RUN_STATUS_FAILED) {
+  console.log('=>(index.tsx:181) runStatus', runStatus)
```
Comment (koji) [inline app/src/pages/RunSummary/index.tsx]: need to keep?

Comment (mjhuff) [reply app/src/pages/RunSummary/index.tsx]: nice catch, thanks

**Review by koji — APPROVED** [review]
the changes look good to me

**Pull request merged**